torchgeo/datamodules/vaihingen.py

```diff
@@ -3,10 +3,14 @@
 
 """Vaihingen datamodule."""
 
-from typing import Any, Dict, Optional
+from typing import Any, Dict, List, Optional, Tuple
 
+import kornia.augmentation as K
 import pytorch_lightning as pl
+import torch
+from einops import repeat
 from torch.utils.data import DataLoader, Dataset
+from torch.utils.data._utils.collate import default_collate
```

Comment:
```suggestion
from torch.utils.data import DataLoader, Dataset, default_collate
```
We shouldn't import from underscore files, those aren't public

Reply:
Not doing that is raising an error during minimum pytests: https://github.com/microsoft/torchgeo/actions/runs/3339526468/jobs/5528583740. `ImportError: cannot import name 'default_collate' from 'torch.utils.data' (/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/torch/utils/data/__init__.py)`

Reply:
Alright, we can change this next time we bump the minimum version of PyTorch. It looks like it isn't public until 1.11.